hydra-node/hydra-node.cabal

```diff
@@ -363,5 +364,5 @@ test-suite tests
     , websockets
     , yaml
 
-  build-tool-depends: hspec-discover:hspec-discover
+  build-tool-depends: hspec-discover:hspec-discover -any
```

Comment:
I can't find any documentation of this `-any` option. What is it for?

Reply:
It was added by my formatting tool and looks like a version indicator?